src/fdc/DiskImageUtils.hh

```diff
@@ -111,27 +135,28 @@ namespace DiskImageUtils {
 	 */
 	void checkValidPartition(SectorAccessibleDisk& disk, unsigned partition);
 
-	/** Like above, but also check whether partition is of type FAT12.
+	/** Like above, but also check whether partition is of type FAT12 or FAT16.
 	 */
-	void checkFAT12Partition(SectorAccessibleDisk& disk, unsigned partition);
+	void checkSupportedPartition(SectorAccessibleDisk& disk, unsigned partition);
 
 	/** Check whether the given disk is partitioned.
 	 */
 	[[nodiscard]] bool hasPartitionTable(SectorAccessibleDisk& disk);
 
 	/** Format the given disk (= a single partition).
 	 * The formatting depends on the size of the image.
-	 * @param disk the disk/partition image to be formatted
-	 * @param dos1 set to true if you want to force dos1 formatting (boot sector)
+	 * @param disk The disk/partition image to be formatted.
+	 * @param bootType The boot sector type to use.
 	 */
-	void format(SectorAccessibleDisk& disk, bool dos1 = false);
+	void format(SectorAccessibleDisk& disk, MSXBootSectorType bootType = MSXBootSectorType::DOS2);
```

Comment:
I know the old code already used a default value. But I think it's better to make `bootType` a mandatory parameter (leave out the default value). I quickly checked and I think all callers do already pass a value.

And the same for `partition()` on line 159.

Reply:
Agreed. Fyi it was called with default argument from `RamDSKDiskImage.cc`.